apps/vs-code-designer/src/app/commands/createNewCodeProject/createCodeProjectSteps/SetWorkspaceName.ts

```diff
@@ -14,7 +14,7 @@ export class setWorkspaceName extends AzureWizardPromptStep<IProjectWizardContex
   public async prompt(context: IProjectWizardContext): Promise<void> {
     context.workspaceName = await context.ui.showInputBox({
       placeHolder: localize('setWorkspaceName', 'Workspace name'),
-      prompt: localize('workspaceNamePrompt', 'Provide a Workspace name'),
+      prompt: localize('workspaceNamePrompt', 'Provide a workspace name'),
```

Comment:
Do these text updates need Esther verification?

Reply:
This was suggested by Esther but did not go through the original PR - see link for original comments:  https://github.com/Azure/LogicAppsUX/pull/2427#discussion_r1207223652